### DISCUSSION TITLE
New Roadmap suggestions

### DISCUSSION THREAD

**Discussion by kaiwalyakoparkar** [commclassroom/roadmaps#27] [General]
Want some new roadmap, suggest them here

**Comment by prateek041** [comment]
### Add: Cyber security Roadmap:

- Cyber security/Computer system security is one of the most distributed domains, having a structure to the learning process will help folks a lot.

**Reply by kaiwalyakoparkar** [reply]
Thank you for your suggestion

**Reply by cosmicqbit** [reply]
**Cyber Security Roadmap** : https://github.com/gokhangokcen1/cyber-security-roadmap

**Comment by Bhushan1701** [comment]
**Add:** Data science Roadmap  (Machine learning, computer vision, Natural language processing,deep learning and many more)
.Data science is an interdisciplinary field that uses scientific methods, processes, algorithms and systems.

**Comment by navyagarwal** [comment]
### Add: ML/DL Roadmap:
- Roadmap, and especially good communities and open source projects in the machine learning, deep learning domain
- Plus, if there could be a little something about the different domains like Computer Vision, Natural Language Processing that branch out from ML/DL
- A relating/ intersecting domain could also be data science

**Reply by cosmicqbit** [reply]
**ML Roadmap** : https://whimsical.com/machine-learning-roadmap-2020-CA7f3ykvXpnJ9Az32vYXva

**Comment by souparnadutta** [comment]
### Add: Data Engineering Roadmap

It would help us to understand all the skills required by following a roadmap and side-by-side guidance/discussion which is rare for this field.

**Reply by cosmicqbit** [reply]
**Data Engineering Roadmap** : https://github.com/datastacktv/data-engineer-roadmap

**Reply by cosmicqbit** [reply]
@Bhushan1701

**Comment by SyedSaifuddin045** [comment]
### ADD : BLOCKCHAIN CORE DEVELOPMENT

**Reply by cosmicqbit** [reply]
Check this out: https://github.com/spo0ds/Journey-to-become-a-Blockchain-Engineer

**Comment by siddhant-khisty** [comment]
**Add:** Game Development Roadmap

**Reply by cosmicqbit** [reply]
**Game Developer Roadmap** : https://github.com/utilForever/game-developer-roadmap

**Comment by abhiram11** [comment]
**ADD: System Design and Cloud Architectures**
We can start with some videos from Gaurav Sen and other youtubers. Once people get familiar with it we can share more content as well.
Here's a great playlist: https://youtube.com/playlist?list=PLhr1KZpdzukdeX8mQ2qO73bg6UKQHYsHb

**Comment by Pritam0077** [comment]
### Add: Cloud solution architect roadmap

**Comment by Lalithsha** [comment]
Add: App development roadmap (Android, IOS, flutter, React native)

**Reply by cosmicqbit** [reply]
**Android Development Roadmap** : https://roadmap.sh/android

**Reply by HGITUP** [reply]
Need Hybrid mobile development roadmap

**Comment by SaptarshiSarkar12** [comment]
# **_Add :_** **Java Full Stack Engineer**

**Reply by cosmicqbit** [reply]
### Java Developer Roadmaps 
- [Roadmap 2021](https://github.com/s4kibs4mi/java-developer-roadmap)
- [Roadmap 2022](https://github.com/devoxx/JavaRoadmap2022)

**Reply by kaiwalyakoparkar** [reply]
Thanks for sharing @cosmicqbit 🙌

**Reply by cosmicqbit** [reply]
> Thanks for sharing @cosmicqbit 🙌

Pleasure to be of help 😊

**Comment by HGITUP** [comment]
Need hybrid mobile development roadmap

**Comment by SaptarshiSarkar12** [comment]
# **_ADD_ : Data Structures And Algorithms Roadmap**

**Reply by seshankgit** [reply]
Yes please add

**Comment by Arghya-Banerjee** [comment]
Please add a **Data Analysis** and **Data Science** Roadmap

**Comment by Yash-Parwani** [comment]
ADD: Augmented Reality/ Virtual Reality roadmap

**Comment by SaptarshiSarkar12** [comment]
**_ADD_ : Database Management System roadmap.**

**Comment by ghost** [comment]
Add : Cyber security roadmap

**Reply by SaptarshiSarkar12** [reply]
#85 This is in progress already and will be updated when completed.

**Comment by vaibhav-vai** [comment]
Add: DSA road map please
